src/selectors/index.js

```diff
@@ -0,0 +1,9 @@
+export const selectFields = ({ id, by, title, kids, url, time, score }) => ({
```

Comment:
What does this function do and what's the purpose of a dedicated file for that?

Reply:
I had removed the usage for this I will clean this up.